js/apps/events-page/constants.js

```diff
@@ -31,7 +31,7 @@
 		requestChunkSize: 10,
 		ageDisclaimer: {
 			message: 'Children under 8 must be accompanied by adult',
-			ageGroupIds: [9, 10, 11, 12]
+			ageGroupIds: [9, 10, 11, 12, 13] // All ages but teen and adults, https://bcpl.evanced.info/api/signup/agegroups
```

Comment:
Thanks for the comment.

Reply:
@ascire i'm not sure, I'll look it up this morning.

Reply:
@ascire from the horses mouth

Reply:
@martypowell interesting.  So I should expect to see changes to package-lock.json *only* when package.json changes.  Otherwise I should treat it as a red flag that something about your environment changed.  Does that sound about right?

Reply:
@ascire seems right to me.